pennylane/devices/default_qubit.py

```diff
@@ -379,6 +380,21 @@ def name(self):
         """The name of the device."""
         return "default.qubit"
 
+    @property
+    def prng_key(self):
+        """Get a new key with ``jax.random.split``."""
+        self._prng_key, key = jax_random_split(self._prng_key)
+        return key
+
+    def get_prng_key(self, num: int = 1):
```

Comment:
Does the first function make sense? I think it might be unexpected that a "property" changes upon each call.

We might also want to make the second function name plural to highlight the distinction with the first.